core/src/avm1/object/bitmap_data.rs

```diff
@@ -12,23 +12,25 @@ use std::fmt;
 #[collect(no_drop)]
 pub struct BitmapDataObject<'gc>(GcCell<'gc, BitmapDataData<'gc>>);
 
-#[derive(Clone, Collect)]
-#[collect(no_drop)]
-pub struct BitmapDataData<'gc> {
-    /// The underlying script object.
-    base: ScriptObject<'gc>,
-    data: GcCell<'gc, BitmapData<'gc>>,
-}
-
 impl fmt::Debug for BitmapDataObject<'_> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let this = self.0.read();
+        let data = this.data.read();
         f.debug_struct("BitmapData")
-            .field("data", &this.data)
+            .field("ptr", &self.0.as_ptr())
+            .field("data", &data)
             .finish()
     }
 }
 
+#[derive(Clone, Collect)]
```

Comment:
Could you move this back to its original location, to shrink the diff?

Reply:
Fixed